graphql/executor.go

```diff
@@ -283,6 +287,14 @@ func (e *Executor) execute(ctx context.Context, typ Type, source interface{}, se
 	switch typ := typ.(type) {
 	case *Scalar:
 		return unwrap(source), nil
+	case *Enum:
+		val := unwrap(source)
```

Comment:
is this unwrap here necessary? I think is. Let's make sure you have a test where a query returns a *Enum.